### PR TITLE
Add scripts/install_plugin.sh to archive

### LINKED ISSUE #130: helm plugin install plugin.yaml metadata missing error
I downloaded the helm-2to3_0.3.1_linux_amd64.tar.gz and unzipped.  When went to install the plugin using helm 3:

helm plugin install

Error: plugin metadata (plugin.yaml) missing

### PR DESCRIPTION
**Pull request by rimusz**
Signed-off-by: rimas <rmocius@gmail.com>

fixes #130

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -16,5 +16,6 @@ archive:
     - LICENSE
     - plugin.yaml
     - completion.yaml
+    - scripts/install_plugin.sh
```
Comment (hickeyma) [inline .goreleaser.yml]: Can we also include "README.md"?

Comment (rimusz) [reply .goreleaser.yml]: sure

Comment (rimusz) [reply .goreleaser.yml]: done

**Comment by rimusz** [comment]
> Thanks for pushing PR @rimusz. Small comment inline. Also, can we move the following files in the install script:
> 
> ```
> mv "releases/v${version}/plugin.yaml" .
> mv "releases/v${version}/README.md" .
> mv "releases/v${version}/LICENSE" 
> ```
> 
> I think we should use the release versions instead from master.

Done

**Pull request merged**